src/resolvers/exotics/git-resolver.js

```diff
@@ -142,7 +142,23 @@ export default class GitResolver extends ExoticResolver {
         return file;
       }
     }
-
-    throw new MessageError(this.reporter.lang('couldntFindManifestIn', transformedUrl));
+    // no registry files were found, in that case we fake a package.json thus allowing leaf dependencies without
+    // any registry file, same as bower
+    // TODO hack
+    const names = parts.path.split('/');
+    const registry = registries.npm;
+    const json = {
+      name: names[names.length - 1],
```

Comment:
@andreypopp Name is available and the syntax is something like
`yarn add foobar@https://github.com/bestander/chrome-app-livereload.git`